src/Sylius/Bundle/ApiBundle/Payment/CompositeApiPaymentMethodInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Payment;
+
+use Sylius\Component\Core\Model\PaymentInterface;
+use Sylius\Component\Core\Model\PaymentMethodInterface;
+
+/** @experimental */
+interface CompositeApiPaymentMethodInterface
```

Comment:
The `Composite` prefix should be in provider service, that iterates over a collection of payment handlers

```suggestion
interface ApiPaymentMethodHandlerInterface
```